### DISCUSSION TITLE
error：Error complete timer task

### DISCUSSION THREAD

**Discussion by Johnite** [uber/cadence#4393] [Q&A]
在查看cadence-server的日志的时候发现了一些error：
{"level":"error","ts":"2021-08-20T11:22:08.019+0800","msg":"Error complete timer task","service":"cadence-history","shard-id":418,"address":"IP:7934","component":"timer-queue-processor","error":"queue processor has been shutdown","logging-call-at":"timer_queue_processor.go:347","stacktrace":"github.com/uber/cadence/common/log/loggerimpl.(*loggerImpl).Error\n\t/Users/didi/go/src/go.uber.org/cadence-0.20.0/common/log/loggerimpl/logger.go:131\ngithub.com/uber/cadence/service/history/queue.(*timerQueueProcessor).completeTimerLoop\n\t/Users/didi/go/src/go.uber.org/cadence-0.20.0/service/history/queue/timer_queue_processor.go:347"}

上面提到的：queue processor has been shutdown，这是什么引起的呢？

**Comment by longquanzheng** [comment]
It could be various reasons. Some of them is not harmful if not constantly happening. 

Could you check other error logs that before this one? There should be some correlated log entries .
